src/components/InRequest/InRequest.tsx

```diff
@@ -1,654 +1,57 @@
-import { ComboBox, DatePicker, IComboBoxOption } from "@fluentui/react";
-import {
-  FunctionComponent,
-  useContext,
-  useEffect,
-  useState,
-  useMemo,
-} from "react";
-import { PeoplePicker, SPPersona } from "../PeoplePicker/PeoplePicker";
-import { OFFICES } from "../../constants/Offices";
-import { GS_GRADES, NH_GRADES, MIL_GRADES } from "../../constants/GradeRanks";
-import { EMPTYPES } from "../../constants/EmpTypes";
-import { WORKLOCATIONS } from "../../constants/WorkLocations";
-import {
-  makeStyles,
-  Button,
-  Input,
-  Text,
-  Label,
-  Radio,
-  RadioGroup,
-  tokens,
-} from "@fluentui/react-components";
+import { FunctionComponent, useContext } from "react";
+import { Button } from "@fluentui/react-components";
 import { UserContext } from "../../providers/UserProvider";
-import { IInRequest, RequestApiConfig } from "../../api/RequestApi";
+import { useRequest } from "../../api/RequestApi";
 import { InRequestViewCompact } from "./InRequestViewCompact";
 import { InRequestEditPanel } from "./InRequestEditPanel";
-import { useForm, Controller, SubmitHandler } from "react-hook-form";
 import { useBoolean } from "@fluentui/react-hooks";
-import { useEmail } from "../../hooks/useEmail";
-
-/**
- * Enum for holding the possible views of the In Request form view
- * @readonly
- * @enum {number}
- */
-export enum INFORMVIEWS {
-  //* Compact view for use within other components, to view details of the In Processing Request */
-  COMPACT,
-  //* Full page view for entering a new In Processing request */
-  NEW,
-  //* Popup/Inline view for editing details of an exisiting In Processing Request */
-  EDIT,
-}
-
-/* FluentUI Styling */
-const useStyles = makeStyles({
-  formContainer: { display: "grid", paddingLeft: "1em", paddingRight: "1em" },
-  floatRight: {
-    float: "right",
-  },
-  errorText: {
-    color: tokens.colorPaletteRedForeground1,
-    fontSize: tokens.fontSizeBase200,
-  },
-});
 
 export const InRequest: FunctionComponent<any> = (props) => {
-  const classes = useStyles();
-  const requestApi = RequestApiConfig.getApi();
   const userContext = useContext(UserContext);
```

Comment:
Since we updated UserApi to now pull straight from a variable rather than a net call, and the user's roles are coming from a react-query that leverages persistent caching after the first request.  What are your thoughts on removing the UserContext all together and just using the User and Role hooks as needed?  Obviously this can just be a new "issue" to work in a separate update.